_translations/en.yml

```diff
@@ -676,9 +676,12 @@ en:
     anonymous: "Bitcoin is not anonymous"
     anonymoustxt: "Some effort is required to protect your privacy with Bitcoin. All Bitcoin transactions are stored publicly and permanently on the network, which means anyone can see the balance and transactions of any Bitcoin address. However, the identity of the user behind an address remains unknown until information is revealed during a purchase or in other circumstances. This is one reason why Bitcoin addresses should only be used once. Always remember that it is your responsibility to adopt good practices in order to protect your privacy. <a href=\"#protect-your-privacy#\"><b>Read more about protecting your privacy</b></a>."
     instant: "Unconfirmed transactions aren't secure"
-    instanttxt: "A Bitcoin transaction is usually deployed within a few seconds and begins to be confirmed in the following 10 minutes. During that time, a transaction can be considered authentic but still reversible. Dishonest users could try to cheat, which means there is a risk when accepting unconfirmated transactions. For large amounts like 1000 US$, it makes sense to wait for 6 confirmations or more. Each confirmation <i>exponentially</i> decreases the risk of a reversed transaction."
+    instanttxt: "New transactions start out as unconfirmed. If you trust the person paying you, you don't have to worry about unconfirmed transactions. If you don't trust them, you should wait for the transaction to receive at least one confirmation—this has historically provided a better than 95% guarantee that the payment can't be changed. Because Bitcoin miners sometimes misbehave, this guarantee isn't as strong for lightweight wallets, so you may want to wait for three confirmations unless you use a recent version of <a href=\"#download#\">Bitcoin Core</a>. For high value amounts, waiting for six or more confirmations is strongly recommended no matter what wallet you use."
```

Comment:
@harding I like it!

For brevity, (I always try to fit in 4 lines, and never exceed 5 lines), how about we merge the first lines together?

Additionally, I think the 10 minutes approximation is pretty useful and important, even if it possible that it might need to be revised in the near future.

Here's an example with these changes:

~~~
New transactions start out as unconfirmed. Unless you trust the person paying you,
you should always wait for the transaction to receive at least one confirmation, which
takes 10 minutes on average. This has historically provided a better than 95% garantee
that the payment can't be changed. This guarantee isn't as strong for lightweight wallets,
so you may want to wait for three confirmations unless you use a recent version of
Bitcoin Core. For high value amounts, waiting for six or more confirmations is strongly
recommended no matter what wallet you use.
~~~